contrib/telegraf/README.md

```diff
@@ -0,0 +1,30 @@
+# backup stats in InfluxDB via Telegraf
+
+Telegraf can ingest stats produced by resticprofile in three different ways:
+
+1. Configure Telegraf to parse resticprofile's JSON status file. The parsing directive looks brittle, but as of June 2024, this seems to be the most viable method.
```

Comment:
Could you please elaborate why the parsing directive looks brittle?

(for information we're not planning on changing the format of the status file anytime soon)

Reply:
It's more of a gut feeling on my side, powered by the amount of time spent on that `(...).@group` solution :)

More seriously though, it does encode expectations how that JSON structure will look like on the consuming side. It's one more place to worry about if that format ever changes.